src/nni_manager/training_service/pai/paiK8S/paiK8STrainingService.ts

```diff
@@ -144,66 +143,60 @@ class PAIK8STrainingService extends PAITrainingService {
             throw new Error('trial config is not initialized');
         }
         const jobName = `nni_exp_${this.experimentId}_trial_${trialJobId}`
-        const paiJobConfig: any = {
-            protocolVersion: 2, 
-            name: jobName,
-            type: 'job',
-            jobRetryCount: 0,
-            prerequisites: [
-              {
-                type: 'dockerimage',
-                uri: this.paiTrialConfig.image,
-                name: 'docker_image_0'
-              }
-            ],
-            taskRoles: {
-                taskrole: {
-                    instances: 1,
-                    completion: {
-                       minFailedInstances: 1,
-                       minSucceededInstances: -1
-                    },
-                    taskRetryCount: 0,
-                    dockerImage: 'docker_image_0',
-                    resourcePerInstance: {
-                        gpu: this.paiTrialConfig.gpuNum,
-                        cpu: this.paiTrialConfig.cpuNum,
-                        memoryMB: this.paiTrialConfig.memoryMB
-                    },
-                    commands: [
-                        command
-                    ]
-                }
-            },
-            extras: {
-                'com.microsoft.pai.runtimeplugin': [
-                    {
-                        plugin: this.paiTrialConfig.paiStoragePlugin
-                    }
+        let paiJobConfig: any = undefined;
+        if (this.paiTrialConfig.paiConfigPath) {
+                paiJobConfig = yaml.safeLoad(fs.readFileSync(this.paiTrialConfig.paiConfigPath, 'utf8'));
```

Comment:
If my paiJobConfig is accidentally changed or deleted during the run of experiment, it will submit inconsistent jobs. RIght?

Reply:
Yes, paiJobConfig file is read when new trial job is created, it may be changed by users.